TheWarWithin/HunterBeastMastery.lua

```diff
@@ -2003,35 +2025,49 @@ spec:RegisterAbilities( {
             if talent.wild_instincts.enabled and buff.call_of_the_wild.up then
                 applyDebuff( "target", "wild_instincts", nil, buff.wild_instincts.stack + 1 )
             end
+
+            if talent.covering_fire.enabled then buff.beast_cleave.expires = buff.beast_cleave.expires + 1 end
```

Comment:
Probably need to check whether Beast Cleave is already up before trying to extend it by 1s.

`if talent.covering_fire.enabled and buff.beast_cleave.up then...`

Reply:
agree